### PR TITLE
Bat safer edits

### PR DESCRIPTION
**Pull request by Mraymon5**
Most of the edits I've been making, minus a few that were more likely to cause downstream issues pending testing.

### CODE REVIEW

File: blech_channel_profile.py
Comment (abuzarmahmood) [inline blech_channel_profile.py:1]: Have you tested if this runs without any metadata? i.e. with a completely unprocessed dataset?

**Comment by abuzarmahmood** [comment]
Most of it looks good. I left just one comment.

**Comment by Mraymon5** [comment]
Yeah; I tried running that on a day that I hadn't done anything else with,
and it worked out fine, as far as I can tell

**Review by abuzarmahmood — APPROVED** [review]
Looks good to me

**Pull request merged**